docs/advanced_documentation/build-guide.md

```diff
@@ -48,7 +48,7 @@ You can define the environment variable `CXX` to for example `clang++` to specif
 
 **Windows**
 
-* MSVC >= 14.2 (Visual Studio 2019, IDE or build tools)
+* MSVC >= 16.2 (Visual Studio 2019, IDE or build tools)
```

Comment:
The documented MSVC version `14.2` cannot have been correct, because MSVC only introduced the first C++17 features in version `15.0`.

C++20 support has been enabled in MSVC version `16.0`. Since we're only using the C++20 STL and no other language features, I believe `16.0` should also do. Do you agree @TonyXiang8787 ?

Reply:
```suggestion
* MSVC >= 16.0 (Visual Studio 2019, IDE or build tools)
```